testsuite/standalone/src/test/java/org/wildfly/extras/creaper/commands/elytron/tls/AddKeyStoreOnlineTest.java

```diff
@@ -0,0 +1,120 @@
+package org.wildfly.extras.creaper.commands.elytron.tls;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.wildfly.extras.creaper.commands.elytron.AbstractElytronOnlineTest;
+import org.wildfly.extras.creaper.commands.security.realms.AddPropertiesAuthentication;
+import org.wildfly.extras.creaper.core.CommandFailedException;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+
+@RunWith(Arquillian.class)
+public class AddKeyStoreOnlineTest extends AbstractElytronOnlineTest{
+
+    private static final String TEST_KEY_STORE_NAME = "CreaperTestKeyStore";
+    private static final String TEST_KEY_STORE_TYPE = "JKS";
+    private static final Address TEST_KEY_STORE_ADDRESS = SUBSYSTEM_ADDRESS.and("key-store", TEST_KEY_STORE_NAME);
+
+    @After
+    public void cleanup() throws Exception {
+        try {
+            ops.removeIfExists(TEST_KEY_STORE_ADDRESS);
+            administration.reloadIfRequired();
+        } finally {
+            client.close();
```

Comment:
Frankly, I haven't seen that before. However, it may make sense here. Please consult with other team member, though.